test/powershell/engine/Cdxml/assets/CimTest/CimTest.cdxml

```diff
@@ -50,7 +50,7 @@
 
     <!-- Set-CimTest -->
       <Cmdlet>
-        <CmdletMetadata Verb="Set" ConfirmImpact="Medium" HelpUri="http://go.microsoft.com/fwlink/?LinkID=241963" Aliases="scimt"/>
+        <CmdletMetadata Verb="Set" ConfirmImpact="Medium" HelpUri="https://go.microsoft.com/fwlink/?LinkID=241963" Aliases="scimt"/>
```

Comment:
It works?

Reply:
With missing slash added `http://technet.microsoft.com/library/jj635727(v=wps.620).aspx` responds `301` Moved Permanently to `http://docs.microsoft.com/en-us/powershell/module/smbshare/set-smbshare?view=winserver2012-ps`

Reply:
Issue created: #8688